src/yaml/vortext/historic-monastery-props-and-mmps.yaml

```diff
@@ -0,0 +1,42 @@
+group: vortext
+name: historic-monastery-mmps
+version: "1"
+subfolder: 180-flora
+info:
+  summary: Historic Monastery MMPs
+  description: |-
+    Made by Krashspeed on request of Vortext, this download contains various monk BATs which will spice up the monastic life in your cities!
+
+    For your convenience 4 different Mayor Mode Ploppables are included, found in 'Historic Monastery MMPs.dat'.
+  author: Vortext, Krashspeed
+  website: https://www.sc4evermore.com/index.php/downloads/download/25-flora-fauna-and-mayor-mode-ploppables/317-vortext-historic-monastery-props-and-mmps
+  images:
+    - https://www.sc4evermore.com/images/jdownloads/screenshots/thumbnails/Historic Monastery MMPs Overview.jpg
+dependencies:
+  - vortext:historic-monastery-props
```

Comment:
I understand. Maybe I went a bit too far with this, but the idea was to avoid adding non-explicitly installed content to the menus as much as possible, and with sc4pac it's easy to manage anyway. For me it feels like the better ux for end users, even though it's a bit annoying to split up all these packages. I'll leave it like this for now I guess.